src/sentry/templates/sentry/emails/reports/body.html

```diff
@@ -228,6 +228,93 @@
 
   </div>
 
+  {% if report.calendar %}
+
+  <style type="text/css">
+    td.spectrum {  /* XXX: td qualifier is a hack */
```

Comment:
Gonna fix this, this is to avoid `text-align` being superceded by [this rule.](https://github.com/getsentry/sentry/blob/538e9e48db81fe9c0da3ed0cb01c73f41faad539/src/sentry/templates/sentry/emails/email-styles.html#L601)